packages/runtime/src/routeGeneration/templates/express/expressTemplateService.ts

```diff
@@ -38,9 +38,9 @@ export class ExpressTemplateService extends TemplateService<ExpressApiHandlerPar
 
   async apiHandler(params: ExpressApiHandlerParameters) {
     const { methodName, controller, response, validatedArgs, successStatus, next } = params;
-    const promise = this.buildPromise(methodName, controller, validatedArgs);
 
     try {
+      const promise = this.buildPromise(methodName, controller, validatedArgs);
       const data = await Promise.resolve(promise);
```

Comment:
is Promise.resolve mandatory : promise ca be waited directly, no? `const data = await promise;`